### ISSUE TITLE
Built-in dialogs should support Right-to-left

### ISSUE THREAD

**Issue by dennisr35** [wixtoolset/issues#5418]
The built-in dialogs seem to exist in left-to-right writing direction only, for example WixUI_Minimal. There already is a translation to Arabic and Hebrew (RTL languages). The built-in dialogs should have a right-to-left version, like WixUI_Minimal_RTL.

**Comment by dennisr35** [comment]
I found out you need to set LeftScroll, RightToLeft, and RightAligned attributes for all controls. I did this by changing the Wix source code. Setting the attributes for dialogs has no effect, you need to set for controls. Still the EULA license text in WelcomeEulaDlg is shown left-aligned. The RightToLeft flag should mirror all controls, so that the bitmap is displayed on the right side.

**Comment by dennisr35** [comment]
I was able to change the Wix source code, to mirror controls on the standard dialogs. (Wix_Minimal). I had to invert the banner bitmaps also.

The problem which remains is that EULA text (WelcomeEulaDlg/LicenseText) is not right-aligned.

**Comment by barnson** [comment]
It shouldn't be necessary to modify the WiX authoring; all the settings should be settable in the .wxl localization files. If not, that's a bug with .wxl support. http://www.joyofsetup.com/2012/07/14/localizing-more-than-strings-in-wix-v3-6/

**Comment by rseanhall** [comment]
We don't have much RTL experience, but Bob will try with English to make sure the wxl supports it.

**Comment by phillHgl** [comment]
Thanks for the meeting recording.  I have friends who can advise on Hebrew, although I have not used wix_UI in my projects.  I have one WixStdBA project.  How involved does a test project need to be (so I have some scope to pitch to them if you want me to ask if they would help.  I don't know if they have the inclination.)

**Comment by barnson** [comment]
This issue is about MSI UI. WixStdBA and ThmUtil probably need more work to support RTL; not sure because I haven't done it before...

**Comment by holonsphere** [comment]
what sort of solutions are outside the scope of the project here?

**Comment by barnson** [comment]
@holonsphere : Please clarify your question.
